### DISCUSSION TITLE
officeR vs 0.6.2 not able to open Word in Window 11 Office 365

### DISCUSSION THREAD

**Discussion by kirklucy** [davidgohel/officer#496] [Q&A]
I am new to officeR and I was able to learn it just a few hours. I am running R (R 4.3.0 GUI 1.79 Big Sur Intel build (8225)) on an Apple macOS Big Sur version 11.7.6 and I wrote a small Word document, 4 pages with images, tables, and graphs. I then sent the Word document to a friend running Windows 10 and Office 2013 version, they were able to open the document. I then sent the same document to another, who is running Windows 11 an MS Office 365, and they were unable to open the document. is there any know issues? I am using MS Office for MAC version 16.72 and the document is okay for me
[Officer.txt](https://github.com/davidgohel/officer/files/11375475/Officer.txt)

**Comment by davidgohel** [comment]
Cross posted here: https://stackoverflow.com/questions/tagged/r

I am not able to reproduce with MS Office 365

**Reply by kirklucy** [reply]
Thanks Mr. Gohel, so you were able to open the file?

**Comment by kirklucy** [comment]
Thanks to all who have helped on this post. I was informed by the Window 11 MS Office 365 user that they restarted their computer and now are able to open and read the document. I suspect this was a Windows issue and not a code or R packages issue.
